pkg/kubelet/util/util.go

```diff
@@ -18,8 +19,25 @@ package util
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/klog"
 )
 
+// poc, tenant name -> api server id
+var Tenant2api map[string]int
+
+func GetTPClient(kubeClients []clientset.Interface, tenant string) clientset.Interface {
+	var client clientset.Interface
+	pick, ok := Tenant2api[tenant]
+	if len(kubeClients) == 1 || tenant[0] <= 'm' || !ok { // !ok to force a retry
```

Comment:
Why do we still need "tenant[0] <= 'm'" here?

Reply:
removed